packages/php/email-editor/tests/integration/Integrations/Core/Renderer/Blocks/Post_Content_Test.php

```
@@ -263,4 +277,69 @@ public function testItRestoresGlobalPostAfterRendering(): void {
+			array(
+				'post_title'   => 'Test Post Content',
+				'post_content' => '<!-- wp:paragraph --><p>This is regular post content.</p><!-- /wp:paragraph -->',
+				'post_status'  => 'publish',
+				'post_type'    => 'post',
+			)
+		);
+
+		$this->assertNotWPError( $post_id, 'Failed to create test post' );
+
+		// Create a block instance.
+		$block = new \WP_Block(
```

PHPMD (2.15.0)

Avoid unused local variables such as '$block'. (undefined)

(UnusedLocalVariable)